google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/FormatOptions.java

```diff
@@ -106,8 +106,8 @@ public static FormatOptions googleSheets() {
   }
 
   /** Default options for PARQUET format. */
-  public static FormatOptions parquet() {
```

Comment:
Keep the return type as is.
Changing it will break existing customers.

Reply:
changed.

should i handle squashing commits myself or is it done at merge time?

Reply:
Thanks @jbeluch. The commits should be updated now.